lib/Runtime/Library/JavascriptBigInt.h

```diff
@@ -55,15 +56,27 @@ namespace Js
 
         virtual BOOL Equals(Var other, BOOL* value, ScriptContext * requestContext) override;
 
+        static digit_t AddDigit(digit_t a, digit_t b, digit_t * carry);
+        static digit_t SubDigit(digit_t a, digit_t b, digit_t * borrow);
+        static digit_t MulDigit(digit_t a, digit_t b, digit_t * high);
+
     private:
         template <typename EncodedChar>
         void InitFromCharDigits(const EncodedChar *prgch, uint32 cch, bool isNegative); // init from char of digits
 
-        bool MulThenAdd(uint32 luMul, uint32 luAdd);
+        bool MulThenAdd(digit_t luMul, digit_t luAdd);
+        static bool IsZero(JavascriptBigInt * pbi);
+        static JavascriptBigInt * AbsoluteIncrement(JavascriptBigInt * pbi);
+        static JavascriptBigInt * AbsoluteDecrement(JavascriptBigInt * pbi);
+        static JavascriptBigInt * Increment(JavascriptBigInt * aValue);
```

Comment:
Could these four methods return void to make it obvious they are an in-place update? Otherwise somebody glancing at the code might have a hard time understanding the subtle but important difference between `Increment(JavascriptBigInt*)` and `Increment(Var)`.